src/constants/constants.js

```diff
@@ -5,17 +5,37 @@ export const URL_PARAMS = {
   SAMPLE_DATE_AFTER: 'sample_date_after',
   SAMPLE_DATE_BEFORE: 'sample_date_before',
   PROJECTS: 'projects',
-  DATA_SHARING: 'dataSharing',
-  METHODS: 'methods',
+  METHOD_DATA_SHARING: 'omitted_method_data_sharing',
 }
 export const COLLECTION_METHODS = [
   {
     name: 'beltfish',
     description: 'Fish Belt',
+    dataSharingOptions: ['bf_all', 'bf_1', 'bf_2', 'bf_3'],
+  },
+  {
+    name: 'colonies_bleached',
+    description: 'Bleaching',
+    dataSharingOptions: ['cb_all', 'cb_1', 'cb_2', 'cb_3'],
+  },
+  {
+    name: 'benthicpit',
+    description: 'Benthic PIT',
+    dataSharingOptions: ['bp_all', 'bp_1', 'bp_2', 'bp_3'],
+  },
+  {
+    name: 'benthiclit',
+    description: 'Benthic LIT',
+    dataSharingOptions: ['bl_all', 'bl_1', 'bl_2', 'bl_3'],
+  },
+  {
+    name: 'quadrat_benthic_percent',
+    description: 'Benthic Photo Quadrat',
+    dataSharingOptions: ['qbp_all', 'qbp_1', 'qbp_2', 'qbp_3'],
+  },
+  {
+    name: 'habitatcomplexity',
+    description: 'Habitat Complexity',
+    dataSharingOptions: ['hc_all', 'hc_1', 'hc_2', 'hc_3'],
```

Comment:
A different approach to this could be creating an object with different keys to represent the names.
e.g.
```
export const COLLECTION_METHODS = {
  beltfish: {
    description: 'Fish Belt',
    dataSharingOptions: ['bf_all', 'bf_1', 'bf_2', 'bf_3'],
  },
  etc
}
```
Then instead of `COLLECTION_METHODS.find((method) => method.name === methodName)` you could do something like `COLLECTION_METHODS['methodName'])`.
That may or may not work for your use case. Either way, I think it's an interesting alternative.

Reply:
I like it better. Just had to update a few functions to support the new data structure